source/math/FloatComplexVectorList.ooc

```diff
@@ -155,7 +163,9 @@ FloatComplexVectorList: class extends VectorList<FloatComplex> {
 		This _fastFourierTransformHelper(input, 0, input count, buffer, 0, result, 0)
 	}
 	createFFTBuffer: static func (inputSize: Int) -> This {
-		This createDefault(This _fastFourierTransformBufferSize(inputSize))
+		result := This new(This _fastFourierTransformBufferSize(inputSize))
+		result _count = inputSize
```

Comment:
Or, aligning with my suggestion below,
```ooc
This getZeros(This _fastFourierTransformBufferSize(inputSize))
```
after renaming `createDefault` to `getZeros` instead of removing it.